README.md

```diff
@@ -91,7 +92,11 @@ The AWS IoT Device Client currently works by default on IoT devices with common
 
 * C++ 11 or higher
 * [CMake](https://cmake.org/) 3.10+
-* *Note* CI Build uses OpenSSL 1.1.1 
+
+### Recommended Requirements
+
+* *Note:* CI Build uses OpenSSL 1.1.1
```

Comment:
Isn't OpenSSL required? I think we show have it under 'Minimum Requirements' section

Reply:
Well technically you could potentially use a different version.  But I see how this could be confusing